ls1intum/Artemis
# `Programming exercises`: Always show result details

### Checklist
#### General


#### Client

### Motivation and Context


#5824 made clear that the current feedback for results is not sufficient.
Students should always be able to take a look at the detailed feedback even if all tests pass.

### Description

Students can now always click on their result and the tasks in the problem statement and see their feedback.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Programming Exercise with some tests after the deadline

1. Log in to Artemis
2. Participate in the exercise with all failing/some failing/all passing tests before the deadline
3. Participate in the exercise with all failing/some failing/all passing tests after the deadline
4. Always make sure that all results show their details, and the UI make everything intuitively clear

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage
